frontend/src/pages/SubProjects/reducer.js

```diff
@@ -71,12 +78,20 @@ const defaultState = fromJS({
   displayNameForPermissions: "",
   showProjectAssignees: false,
   projectAssignee: "",
-  dialogTitle: strings.subproject.subproject_add_title
+  dialogTitle: strings.subproject.subproject_add_title,
+  searchTerm: "",
+  searchTerms: [],
+  searchBarDisplayed: false,
+  highlightingRegex: ""
 });
 
 export default function detailviewReducer(state = defaultState, action) {
   switch (action.type) {
     case FETCH_ALL_PROJECT_DETAILS_SUCCESS:
+      // While searching, stop updating the project list from new fetched projects
+      if (state.get("searchTerms").size === 0) {
+        state = state.set("filteredSubProjects", action.subprojects);
```

Comment:
subprojects shouldn't change if projects are fetched successfully.

Reply:
what were the actions on this comment?

Reply:
i dont understand. PROJECT_DETAILS refers to the details of a projects, which are the subrpojects (and SUBPROJECT_DETAILS refer to the workflow items)

Reply:
you're right, so please update the comment since it leads to confusion (to subproject list and fetched subprojects )

Reply:
this is already done, see: https://github.com/openkfw/TruBudget/blob/sub_search/frontend/src/pages/SubProjects/reducer.js